sql/sys_vars.cc

```diff
@@ -6071,6 +6071,13 @@ static Sys_var_plugin Sys_default_tmp_storage_engine(
     MYSQL_STORAGE_ENGINE_PLUGIN, DEFAULT(&default_tmp_storage_engine),
     NO_MUTEX_GUARD, NOT_IN_BINLOG, ON_CHECK(check_storage_engine));
 
+static Sys_var_plugin Sys_default_dd_storage_engine(
```

Comment:
Since this is different from other "default SE" sysvars in that each allowed value is handled specifically, I'd consider making it enum, not string.